src/apps/frontend/pages/authentication/forgot-password/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import toast from 'react-hot-toast';
+
+import { PASSWORD_RESEND_DELAY_IN_MILLISECONDS } from '../../../constants/timer';
+import { AsyncError } from '../../../types';
+import useTimer from '../../../utils/use-timer';
+import AuthenticationFormLayout from '../authentication-form-layout';
+import AuthenticationPageLayout from '../authentication-page-layout';
+
+import ForgotPasswordForm from './forgot-password-form';
+import ForgotPasswordResendEmail from './forgot-password-resend-email';
+
+export const ForgotPassword: React.FC = () => {
+  const [isResendEmailPage, setIsResendEmailPage] = useState<boolean>(false);
```

Comment:
No need to declare state type if we initialize the state value
`const [isResendEmailPage, setIsResendEmailPage] = useState(false);` --> now state type is boolean
`const [username, setUsername] = useState('');` --> now state type is string